### ISSUE TITLE
Distributed's Adaptive's Scheduler `retire_workers` and Cluster `scale_down`

### ISSUE THREAD

**Issue by jakirkham** [dask/distributed#1951]
It appears that [Distributed's Adaptive tells the Scheduler to `retire_workers` and then `scale_down` the Cluster]( https://github.com/dask/distributed/blob/1.21.7/distributed/deploy/adaptive.py#L230-L235 ). Is there a reason both of these operations are needed? Also what does this mean for downstream libraries inheriting from Distributed's Adaptive or Distributed's Cluster?

xref: https://github.com/dask/dask-drmaa/issues/65 (for context)

**Comment by mrocklin** [comment]
Scale down is used here as a method of last resort to kill jobs after they
should already have been closed down smoothly.  We don't have to do this
though.  Retiring along is probably sufficient.

**Comment by rbubley** [comment]
I had understood the intended semantics to be different: `retire_workers` is for the Scheduler to tell Worker/Nanny processes to terminate; `scale_down` is at the cluster level, to tell a machine to shut down. Is this not the intention?

**Comment by mrocklin** [comment]
Yes, retire_workers tells the worker and nanny processes to terminate.
Typically this results in the cluster-controlled job also being marked as
finished at the resource manager level (kubernetes, yarn, sge, ...)

scale_down tells the resource manager (kubernetes, yarn, sge, ...) to
terminate the job.

Nothing is able to tell the machine itself to shut down (I'm not sure if
this was your intention).  This typically isn't available to anyone except
administrators at the physical location.

**Comment by jakirkham** [comment]
This might be a silly question, but how do these compare to `stop_worker`/`stop_workers`?

**Comment by mrocklin** [comment]
stop_worker and stop_workers were older terms.  We might consider
deprecating them

**Comment by jakirkham** [comment]
SGTM

**Comment by jakirkham** [comment]
How does this relate to `start_workers` and `scale_up`?

**Comment by mrocklin** [comment]
start workers is analagous to stop_workers.  We might consider deprecating them.  scale_up is currently used as part of the scale/up/down trio

**Comment by GenevieveBuckley** [comment]
@jakirkham do you think more information needs to be added to the docs, or was this question more for your own personal understanding?
